src/etc/inc/interfaces.inc

```diff
@@ -2357,6 +2358,73 @@ function interface_virtual_create($interface)
     }
 }
 
+function interface_dslite_configure($verbose, $parent = 'wan', $aftr)
+{
+    global $config;
+
+    // Should be tested outside, but let's make sure we won't mess around
+    if (!interface_dslite_is_enabled($parent)) {
+        return;
+    }
+
+    $ip = get_interface_ipv6($parent);
+
+    if ($verbose) {
+        log_error(sprintf("Starting DS-Lite tunnel with AFTR '%s' on interface %s with external IP '%s'", $aftr, $parent, $ip));
+    }
+
+    $gifif = legacy_interface_create('gif', "{$parent}_dslite");
+
+    $mtu = '';
+    if (!empty($config['interfaces'][$parent]['dslite_tunnel_mtu'])) {
+        $mtu = sprintf('mtu %s', $config['interfaces'][$parent]['dslite_tunnel_mtu']);
```

Comment:
This should be:

$mtu = exec_safe('mtu %s ', $config['interfaces'][$parent]['dslite_tunnel_mtu']);